packages/fxa-content-server/app/tests/spec/views/settings.js

```diff
@@ -131,8 +136,90 @@ describe('views/settings', function() {
     assert.equal(args[1], 'wibble');
   });
 
+  describe('setInitialContext:', () => {
```

Comment:
Ahha, this is why the invalid variable name wasn't caught, the full render is never done with it in the unit tests.